core/blockchain.go

```diff
@@ -399,6 +407,7 @@ func (bc *BlockChain) SetHead(head uint64) error {
 			}
 			rawdb.WriteHeadFastBlockHash(db, newHeadFastBlock.Hash())
 			bc.currentFastBlock.Store(newHeadFastBlock)
+			headReceiptGauge.Update(int64(newHeadFastBlock.NumberU64()))
```

Comment:
Wouldn't it be more consistent with earlier naming to have the meter called `headFastBlockGauge` instead of `headReceiptGauge` ?

Reply:
Fair enough!